src/modules/olMap/utils/olMapUtils.js

```diff
@@ -1,3 +1,5 @@
+import { OlMap } from '../components/OlMap';
```

Comment:
can be removed...
although referenced for jsdoc, `npm run doc` does not throw an error, if removed

Reply:
Oops, thx

Reply:
Hmm, seems to work locally and here on CI